src/platform/StdSchedulerWin32.cpp

```diff
@@ -87,9 +87,8 @@ bool StdScheduler::DoScheduleProcs(int iTimeout)
 	// Iterate over the index because procedures may be added or removed during execution
 	// (If they are removed, we skip one execution, which doesn't really matter in practice)
 	auto tNow = C4TimeMilliseconds::Now();
-	for (size_t i_proc = 0u; i_proc < procs.size(); ++i_proc)
```

Comment:
I don't know if a schedule proc ever removes itself, but I find it very likely (if not, it may very well happen in the future).

I favor the indexed version (on a vector) because it's simple and it works no matter whether a schedule proc removes itself or another proc. The only downside is that removal offsets the index by one, so we miss execution of the following proc once. This should not matter in practice.

The specific loop did cause a crash in the past.